.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -13,16 +17,30 @@ Reviewing guidelines: https://hydra.nixos.org/job/nixpkgs/trunk/manual/latest/do
 
 ###### Things done
 
-<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->
+<!-- 
+Requirements: 
+- install `nixpkgs-review` for good with `nix-env -f '<nixpkgs>' -iA nixpkgs-review`
+- consult usage documentation: https://github.com/Mic92/nixpkgs-review#usage
+- setup github api token: https://github.com/Mic92/nixpkgs-review#github-api-token
+-->
+
+- [ ] Tested sucessful build of final PR `GITHUB_TOKEN=<YOUR_TOKEN> nixpkgs-review pr <pr-number>`.
+      If suceeded, within the resulting `nix-shell`:
+   - [ ] Manually tested execution of all binary files (in `./results/bin/`)
+   - [ ] Included manual checks and validations at the end of `editor ./report.md`
+   - [ ] If ok, posted the results: `nix-shell> nixpkgs-review post-result`
 
-- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
-- Built on platform(s)
+- Platform(s), I built on (reviewers, please complement!):
+<!-- more is better, reviewers might complement -->
    - [ ] NixOS
    - [ ] macOS
-   - [ ] other Linux distributions
-- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
-- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
-- [ ] Tested execution of all binary files (usually in `./result/bin/`)
-- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
-- [ ] Ensured that relevant documentation is up to date
+   - [ ] other Linux (Ubuntu, Archlinux, Alpine, etc.).
+
+- [ ] If available: tested via one or more NixOS test(s) `GITHUB_TOKEN=<YOUR_TOKEN> nixpkgs-review pr -p nixosTests.<test> <package> <pr-number>` (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests)
```

Comment:
We might also want to mention `passthru.tests`. Or implement https://github.com/Mic92/nixpkgs-review/issues/77 and add the mapping to every package with NixOS test.

Reply:
Could you make a suggestion until the mentioned PR is merged, or should we just be pationt until then and leave this as it is?

Reply:
Maybe we could add a comment like “look for `passthru.tests` within the package expression” or “try running `nix-build -A <package>.tests`”. Or just ignore it for now, since ofborg will already run `passthru.tests` so there is not much need to run them locally. I guess that is fine, no need to ask reviewers to do what machines can.